src/index.html

```diff
@@ -5,8 +5,398 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__link" href="#home">
+        <img
+          src="images/logo.png"
+          alt="Moyo-logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link"  href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link"  href="#">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              data-qa="nav-hover"
+              class="nav__link"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link"  href="#">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a  class="nav__link" href="#">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link"  href="#">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link"  href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__content">
+        <article class="card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="Apple iMac 27&quot; Retina 5k Monoblock"
+            class="card__image"
+          >
+
+          <div class="card__nameblock">
```

Comment:
```suggestion
             <div class="card__name-block">
```

Check please this

Reply:
why then did mentor aprove card catalog if it had the same class names?